client/js/main.js

```diff
@@ -5,14 +5,29 @@ import store from './store/store';
 import router from './router';
 Vue.config.productionTip = false;
 
-const app = new Vue({
-  vuetify,
-  store,
-  router,
-  render: (h) => h(App),
-}).$mount('#app');
+async function initApp() {
+  // Make sure we try and reestablish a player's session if one exists
+  // We do this before the app mounts to preempt the reauth/logout logic
```

Comment:
The comment says that we run this check before mount, but `initApp()` is called below the application mount as the last line of the file. What's the intention here?